test/cljs/cider/nrepl/middleware/cljs_ns_test.clj

```diff
@@ -47,9 +47,11 @@
              {:arglists "(quote ([e]))"}))))
 
   (testing "ns-path op"
-    (let [{:keys [^String path]} (session/message {:op "ns-path"
-                                                   :ns "cljs.core"})]
-      (is (.endsWith path "cljs/core.cljs")))
+    (let [{:keys [^String path ^String url]} (session/message {:op "ns-path"
```

Comment:
Probably you want to update the Clojure tests for this op as well. Seems they are in a different file.

Reply:
See `test/clj/cider/nrepl/middleware/ns_test.clj` in PR diff